Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.yml

```diff
@@ -6431,9 +6427,34 @@ script:
       isArray: true
     description: Remove tags from the indicators.
     name: threatstream-remove-indicator-tag
+  - name: threatstream-clone-imported-indicator
+    arguments:
+    - name: indicator_id
+      required: true
+      description: ID of the indicator to clone.
+    outputs:
+    - contextPath: ThreatStream.Clone.ID
```

Comment:
chamge it also in the README file
```suggestion
    - contextPath: ThreatStream.Clone.indicator_id
```

Reply:
keeping consistent with other anomali calls that return the ID of the indicator as ID vs. the input which is also consistently indicator_id.

Reply:
but in line 2634 you added this to the context under `indicator_id`:
```
    # append the indocator ID being used in the clone
    res_json["indicator_id"] = indicator_id
```

so maybe change this to :
```
    res_json["ID"] = indicator_id
```